lib/ansible/modules/network/cli/cli_config.py

```diff
@@ -184,7 +183,7 @@
 
 RETURN = """
 commands:
-  description: The set of commands that will be pushed to the remote device
+  description: The set of commands that will be pushed to the remote device. Note: The commands will be returned only for platforms that have C(supports_onbox_diff) set to I(false). For other platforms, using the C(--diff) option with the playbook will return the difference in configuration.                
```

Comment:
The Note section should be on the next line which follows the indentation of description section.

Reply:
Should it be 'Note:' or  'note:' ? Because all section keys are lower cased. Will this affect how the template is processed?